spec/rubocop/cop/style/commented_keyword_spec.rb

```diff
@@ -237,6 +237,13 @@ def x(y: "#value")
     RUBY
   end
 
+  it 'does not register an offense for RBS::Inline annotation for subclass definition' do
+    expect_no_offenses(<<~RUBY)
+      class X < Y #[String]
+      end
+    RUBY
+  end
```

Comment:
Can you also add a test case for registering an offense? For example, a class definition without inheritance could be considered.

```ruby
class X #[String]
end
```

Reply:
@koic 

Thanks for your comment!
I added test case for registering an offense to subclass definitioin.